include/k4a/k4atypes.h

```diff
@@ -420,6 +442,23 @@ typedef enum
     K4A_IMAGE_FORMAT_CUSTOM,
 } k4a_image_format_t;
 
+/** Transformation interpolation type.
+ *
+ * \remarks
+ * Interpolation type used with k4a_transformation_custom_depth_image_to_color_camera.
+ *
+ * \xmlonly
+ * <requirements>
+ *   <requirement name="Header">k4atypes.h (include k4a/k4a.h)</requirement>
+ * </requirements>
+ * \endxmlonly
+ */
+typedef enum
+{
+    K4A_TRANSFORMATION_INTERPOLATION_TYPE_NEAREST = 0, /**< Nearest neighbor interpolation */
+    K4A_TRANSFORMATION_INTERPOLATION_TYPE_LINEAR,      /**< Linear interpolation */
+} k4a_transformation_interpolation_type_t;
```

Comment:
will deloader_transform_engine_process_frame ever get called where this parameter will be a don't care? If so we might want to add XXX_NONE.

Reply:
So I originally had a "don't care" option called INTERPOLATION_TYPE_ANY here. It was removed after the first review since it was decided that the client of k4a should always select an interpolation method. You're right that deloader_transform_engine_process_frame could in specific situations use the "don't care" option but that's an internal detail that we probably don't want to expose to the client of k4a. What I could do is add back the "don't care" option to the k4aplugin.h header - but then it's not in sync with the k4a.h enum. @mbleyer Do you have an opinion about this?

Reply:
I will leave this as is for now.